JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -71,7 +71,7 @@ module Microsoft.ApplicationInsights.Context {
                 this.initializeAutomaticSession();
             }
 
-            var now = +new Date;
+            var now = dateTime.Now();
```

Comment:
Yes, it's a wrapper for performance.now, which also checks if the function is supported. See [ajaxUtils.ts](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK/ajax/ajaxUtils.ts#L32)